src/frontend-samples/issues-sample/Issues.scss

```diff
@@ -23,6 +23,20 @@
 .issue-list-header {
     padding: 6px;
     border-bottom: 1px solid $buic-background-divider;
+    display: flex;
+    flex-direction: column;
```

Comment:
Could we change flex-direction here to "row" and maybe add some margin? 
I think there is a lot of wasted space in the widget and would benefit if they were in the same line.

![image](https://user-images.githubusercontent.com/39310001/130121415-1a1628da-d163-4624-ad21-c96c9120e17c.png)

Reply:
On second thought, default size is pretty tiny so might just be okay if they're on separate lines.